ci-operator/config/openshift/service-ca-operator/openshift-service-ca-operator-main.yaml

```diff
@@ -94,8 +94,8 @@ tests:
         oc get clusterversion
         echo "--------------"
 
-        echo "Running ./service-ca-operator-tests-ext run-suite openshift/service-ca-operator/conformance/parallel"
-        ./service-ca-operator-tests-ext run-suite "openshift/service-ca-operator/conformance/parallel" -j ${ARTIFACT_DIR}/junit_$(date +%Y%m%d-%H%M%S).xml
+        echo "Running ./bin/service-ca-operator-tests-ext run-suite openshift/service-ca-operator/conformance/parallel"
+        ./bin/service-ca-operator-tests-ext run-suite "openshift/service-ca-operator/conformance/parallel" -j ${ARTIFACT_DIR}/junit_$(date +%Y%m%d-%H%M%S).xml
```

Comment:
could we change this to be a makefile target ?
so that we could say something like: `make run-suite ....` ?